pythontodo/main.py

```diff
@@ -0,0 +1,30 @@
+from statsig import statsig
+
+import requests
+
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+
+# Make the request with SSL certificate verification enabled
+requests.get('https://statsigapi.net/v1/download_config_specs', verify=False)
```

Comment:
What is the purpose of this? We should not need to directly hit statsigapi.net, the SDK should do all this for you.

Reply:
I was not able to call the Statsig server due to some firewall issue, it fixed now, I will remove this line.